getRN.js

```diff
@@ -0,0 +1,6 @@
+const getRandomNumber = (min = 1, max = 5) => {
+  const number = Math.floor(Math.random() * (max - min)) + min;
+  return console.log(number);
+}
+
+module.exports = (from, to) => () => getRandomNumber(from, to);
```

Comment:
any reason to use here function that returns function?

Reply:
because you write same code in class